app/static/js/yara_rule/yara_rule-controller.js

```diff
@@ -459,6 +459,9 @@ angular.module('ThreatKB')
                         }, function (error) {
                             growl.error(error, {ttl: -1})
                         })
+                    } else if (id_or_rule.reverted) {
+                        growl.info("Successfully reverted '" + $scope.yara_rule.name + "' to revision '" + id_or_rule.revertedToRevision + "'", {ttl: 3000});
+                        getPage();
```

Comment:
Decided to treat this the same way as "Save". When it's reverted, the modal is closed and main page is displayed.